apps/web/src/components/layouts/map-view/components/tree-view.tsx

```
@@ -0,0 +1,199 @@
+  const [items, setItems] = useState<Record<string, Item>>();
+
+  const tree = useTree<Item>({
+    state: treeState,
+    setState: setTreeState,
+    indent,
+    rootItemId: "root",
+    getItemName: (item) => item.getItemData().name,
+    isItemFolder: (item) => item.getItemData().isFolder ?? false,
+    dataLoader: {
+      getItem: (itemId) =>
+        (items ?? {})[itemId] ??
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)